parser/src/Parse/Module.hs

```diff
@@ -96,27 +96,29 @@ import' =
           (_, postAs) <- whitespace
           (,) preAs <$> (,) postAs <$> capVar <?> ("an alias for module `" ++ moduleName ++ "`")
 
-    exposing :: IParser (Comments, PreCommented (Var.Listing Var.Value))
+    exposing :: IParser (Comments, PreCommented (Var.Listing Var.Value), Bool)
     exposing =
       do  (_, preExposing) <- try (whitespace <* reserved "exposing")
           (_, postExposing) <- whitespace
-          (postExposing2, listing') <- listing value
-          return (preExposing, (postExposing ++ postExposing2, listing'))
+          (postExposing2, listing', multiline) <- listing value
+          return (preExposing, (postExposing ++ postExposing2, listing'), multiline)
 
 
-listing :: IParser a -> IParser (Comments, Var.Listing a)
+listing :: IParser a -> IParser (Comments, Var.Listing a, Bool)
 listing item =
   expecting "a listing of values and types to expose, like (..)" $
   do  (_, preParen) <- try (whitespace <* char '(')
       (_, pre) <- whitespace
+      pushNewlineContext
```

Comment:
To be consistent with the rest of elm-format, I think pushNewlineContext should happen a line earlier (before the whitespace).  I will make that change and add a commit.